.github/libs/GitUtils.ts

```diff
@@ -191,24 +103,14 @@ function getValidMergedPRs(commits: CommitType[]): number[] {
  * Takes in two git tags and returns a list of PR numbers of all PRs merged between those two tags
  */
 async function getPullRequestsDeployedBetween(fromTag: string, toTag: string) {
-    console.log(`Looking for commits made between ${fromTag} and ${toTag}`);
-
     const apiCommitList = await GithubUtils.getCommitHistoryBetweenTags(fromTag, toTag);
```

Comment:
NAB but with only these variables remaining the `api` prefix feels unnecessarily verbose.

Reply:
oh 100%, thanks I missed that.